julia/deps/cpcblas.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# be invoked from build.jl
```

Comment:
What does this script do?

Reply:
It's a little bit tricky way to make Travis's cache system happy.
If we cp `cblash.h` to source dir, `make` will rebuild all stuffs and cause Travis timeout and the cache become useless.

Reply:
Ah I see, so this can be removed if we move away from Travis?

Reply:
yeah.